### PR TITLE
Spring Boot Properties Prefix Must Be in Canonical Form

### PR DESCRIPTION
**Pull request by zahikhan**

### CODE REVIEW

File: spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.baeldung.error.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
```
Comment (vinipx) [inline spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java:6]: You don't need these comments here.

File: spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * <h2>This class is for Configuration.</h2>
+ * <p>
+ * Project Name: New folder
+ *
+ * @author Zahid Khan
+ * @version 4/1/2023
+ */
+@Configuration
+@ConfigurationProperties(prefix = "profile.application-properties")
```
Comment (vinipx) [inline spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java:15]: prefix value is different than the one mentioned in the draft: 
https://drafts.baeldung.com/?p=159983&preview=true#basic-configuration

File: spring-boot-modules/spring-boot-properties-3/src/main/resources/application.properties [hunk 3]
```diff
@@ -31,4 +31,13 @@ bael.property=prodValue
...
+baeldung.presentation=${HELLO_BAELDUNG}. Java is installed in the folder: ${JAVA_HOME}
+#---
+#camelCasing
+#profile.applicationProperties.name="Baeldung"
+#PascalCasing
+#profile.ApplicationProperties.name="Baeldung"
+#SnakingCasing
+#profile.application_properties.name="Baeldung"
+#KebabCasing
+profile.application-properties.name="Baeldung"
```
Comment (vinipx) [inline spring-boot-modules/spring-boot-properties-3/src/main/resources/application.properties:43]: profile.**application-**properties.name differs from the one mentioned in the draft:
https://drafts.baeldung.com/?p=159983&preview=true#basic-configuration

**Review by vinipx — CHANGES_REQUESTED** [review]

File: spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java (hunk 1 above)
Comment (zahikhan) [reply spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java]: Remove the comments.

File: spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java (hunk 2 above)
Comment (zahikhan) [reply spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java]: Updated the prefix and draft to keep in sync.

File: spring-boot-modules/spring-boot-properties-3/src/main/resources/application.properties (hunk 3 above)
Comment (zahikhan) [reply spring-boot-modules/spring-boot-properties-3/src/main/resources/application.properties:43]: Resolved this.

**Review by vinipx — APPROVED** [review]

**Comment by vinipx** [comment]
Cool, thanks!

**Pull request merged**